src/server/items/controllers.js

```diff
@@ -93,5 +93,80 @@ export default {
       }
       return cb();
     });
+  },
+  add: (req, res, cb) => {
+    db.get('itemIncrement', (getErr, value) => {
+      if (getErr) {
+        res.status(500);
+        res.send({
+          error: getErr
+        });
+        cb();
+      } else {
+        const increment = value + 1;
```

Comment:
Why do you need this incremental value here? https://www.quora.com/How-do-you-generate-monotonically-increasing-sequence-numbers-seqnums-in-a-large-distributed-system